Kudu.Core/Scripts/starter.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
This complains when bash is not the default shell (e.g Ubuntu 16 uses dash, Alpine Linux uses ash, NetBSD has csh etc.) 
Please consider using more portable shebang: `#!/usr/bin/env bash` instead of `#!/bin/bash`.